Sources/FigmaExport/Input/FigmaExportOptions.swift

```diff
@@ -0,0 +1,35 @@
+import ArgumentParser
+import Foundation
+import Yams
+
+struct FigmaExportOptions: ParsableArguments {
+    static let input = "figma-export.yaml"
+
+    @Option(name: .shortAndLong, help: "An input YAML file with figma and platform properties.")
+    var input: String = Self.input
+
+    var accessToken: String = ""
+
+    var params = Params.placeholder
+
+    var unpack: (String, Params) {
```

Comment:
Why did you add `unpack` property? We can get access to `accessToken` and `params` directly from the `ExportImages`, `ExportColors`, etc.

Reply:
I made `unpack` so as not to change the rest of the code in the commands. I'll redo it